### ISSUE TITLE
[Question] Can not find other's media with Search

### ISSUE THREAD

**Issue by nalyk** [superdesk/superdesk#2950]
Maybe it is a settings related question, but I cannot solve this:

Using an older version of Superdesk, when someone else added some media (pictures, videos) I could always find them with Search. I recently noticed that Superdesk 55477ac that I use in production on one project, is showing while searching, **only text items**.

Could you point me to the solution?

**Comment by petrjasek** [comment]
are there any updates done on the media items? I think the initial version is only visible to the author, and once there are some changes saved it's visible to others, same like text items

**Comment by nalyk** [comment]
@petrjasek you are right. I edited one media item, and it became visible to others. Is it possible to make it visible by default, only by uploading it (to a desk or directly, to article)?

**Comment by nalyk** [comment]
@petrjasek if it not possible to make it visible by default, I would like to simplify the job of journalists and make a minimal template for editing video and photo items. To do so, I must:

1. Add [SCHEMA](http://superdesk.readthedocs.io/en/latest/settings.html#schema) config for an item of type picture to the env.sh (or activate.sh) in the server folder.
2. Add [EDITOR](http://superdesk.readthedocs.io/en/latest/settings.html#editor) config for an item of type picture to the env.sh (or activate.sh) in the server folder.
3. Add [SCHEMA](http://superdesk.readthedocs.io/en/latest/settings.html#schema) config for an item of type video to the env.sh (or activate.sh) in the server folder.
4. Add [EDITOR](http://superdesk.readthedocs.io/en/latest/settings.html#editor) config for an item of type video to the env.sh (or activate.sh) in the server folder.
5. Save and restart Superdesk

Please correct me on this steps (note that in the documentation, SCHEMA editing is only for item types text, picture, and composite).

Thank you very much for your help.
